docs/release/release_0_2_1.rst

```diff
@@ -0,0 +1,77 @@
+Announcement: napari 0.2.1
+==========================
+
+We're happy to announce the release of napari v0.2.1!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-base
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+New Features
+------------
+- added a code of conduct
```

Comment:
We typically put PR numbers in the scikit-image release notes... Are you not using the script for these?

Reply:
Oh, I see that they are below. Typically I move rather than copy PRs from the list to the sections. Not a big deal though.

Reply:
that's a good idea, have changed